CMakeLists.txt

```diff
@@ -291,8 +293,8 @@ IF(MSVC_IDE)
     MESSAGE(STATUS "Tuning for VisualStudio IDE:")
 
     IF(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-        SET(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/ide"
-            CACHE PATH "Install directory used by INSTALL target" FORCE)
+        SET(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/moneymanagerex"
+            CACHE PATH "Install directory used by INSTALL target\nChange 'moneymanagerex' to 'src/Release' or 'src/Debug' for use in IDE" FORCE)
```

Comment:
Don't do it. It's fixed already in my `update-cmake2` branch.
Now it's as easy as:
1. File -> Open directory...
2. CMake -> Install
3. CMake -> Debug

I'm working on the BUILD.md (testing on my VS fresh install) to prepare PR for this.